### DISCUSSION TITLE
Starting Details for User Account and Sessions (Sign In/Log In) Team

### DISCUSSION THREAD

**Discussion by evnd04** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#6] [Informational]
User Account and Sessions (Sign In/Log In) Team responsibilities:

> Successful login
> Authentication, session management, and error responses.
> Buttons, input fields, and error messages.
> Incorrect password handling
> Account lockout after multiple failed attempts.
> Use UPRM email to ensure they’re UPRM students and not outsiders
> Account Creation (Validate e-mail)
> User’s profile (Student)

Provide your name, GitHub username, institutional email to be a part of this group, if you are interested in being a team leader add (team leader) to your info.

Updated Team:

Team leader: @shenied-maldonado 

Developers: 
@joel-alvarado 
@johnsantiago9 
@noel-vargas 
@williammartinez10 
@JuanJRivera 

This team is currently *FULL*

**Comment by shenied-maldonado** [comment]
Shenied Maldonado
shenied-maldonado
shenied.maldonado@upr.edu
(team leader)

**Comment by joel-alvarado** [comment]
Joel Alvarado
joel-alvarado
joel.alvarado2@upr.edu

**Comment by johnsantiago9** [comment]
John R Santiago
Johnsantiago9
John.santiago9@upr.edu

**Comment by noel-vargas** [comment]
Noel A Vargas Padilla
noel-vargas
Noel.vargas5@upr.edu

**Comment by williammartinez10** [comment]
William Martínez
williammartinez10
william.martinez10@upr.edu

**Comment by JuanJRivera** [comment]
Juan J Rivera Mercado 
Juan J Rivera
Juan.rivera103@upr.edu

**Comment by RaquelQuinones** [comment]
Raquel M. Quinones Velez
RaquelQuinones
raquel.quinones7@upr.edu

**Comment by YanehRuiz** [comment]
The team is currently *FULL*

**Comment by lex939** [comment]
**Message for: @joel-alvarado, @johnsantiago9, @noel-vargas, @williammartinez10, @JuanJRivera, and @shenied-maldonado**

Hi team!

Welcome to SafeRUM! I’m excited to work with you all on this project. 

From what I’ve read in the discussion so far, @shenied-maldonado will be the team leader. I need two of the other members in the team to take on the responsibility of handling both documentation and software development for user interface. These two members will split their time 50-50 between documentation and development, so each will do a bit less of each task to collectively meet the same total of tasks per milestone.

Please discuss among yourselves and let me know which people the team decides will fill these roles. Thanks so much!

**Reply by shenied-maldonado** [reply]
Hey @lex939!

We finally decided that we'll have @johnsantiago9 and @JuanJRivera splitting their attention between documentation and software development. If there are any further discussions to be had, feel free to message any of us three about it! 

Thank you!
